packages/gatsby-theme-store/src/sdk/region/setRegion.ts

```diff
@@ -50,41 +50,7 @@ export const mutation = gql`
       }
       orderFormId: $orderFormId
     ) {
-      id
-      shipping {
-        availableAddresses {
-          postalCode
-          receiverName
-          reference
-          state
-          street
-          number
-          neighborhood
-          isDisposable
-          geoCoordinates
-          country
-          complement
-          city
-          addressType
-          addressId
-        }
-        selectedAddress {
-          addressId
-          addressType
-          city
-          complement
-          country
-          geoCoordinates
-          isDisposable
-          neighborhood
-          postalCode
-          number
-          receiverName
-          reference
-          state
-          street
-        }
-      }
+      ...OrderFormFragment_orderForm
```

Comment:
Where does it comes from? i'm curious because I'm not sure how it works hahah

Reply:
Oh, sorry. My bad. It comes from [here](https://github.com/vtex/faststore/blob/master/packages/gatsby-theme-store/src/sdk/orderForm/controller/orderForm.ts#L16-L282)

Reply:
Nice! But I'm thinking if we do need all those data from OrderFormFragment. We could just add the `availability` property. What do you think?

Reply:
The problem is that it can cause inconsistency, because I think, for example, that prices can change depending on the region... I'm not saying that this is the current scenario, but it seems to me that it's better to update everything and avoid having data that does not match. Does it make sense?